### PR TITLE
feat(tutorial): complete step 3

### PR DESCRIPTION
**Pull request by Ada-siqiliu**

### CODE REVIEW

File: src/views/RepoPage/RepoTable.vue [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
-        >
-        <template slot="expandedContent">{{ row.description }}</template>
-      </cv-data-table-row>
-    </template>
-  </cv-data-table>
+    <cv-data-table-skeleton v-if="loading" :columns="columns" :title="title" :helper-text="helperText" :rows="10" />
+    <cv-data-table v-else :columns="columns" :title="title" :helper-text="helperText"
+        :pagination="{ numberOfItems: this.totalRows }" @pagination="$emit('pagination', $event)">
+        <template slot="data">
+            <cv-data-table-row v-for="row in data" :key="`${row.description}`">
```
Comment (ZrianinaMariia) [inline src/views/RepoPage/RepoTable.vue]: I am not sure that it is a good idea to use description as a key.

**Comment by ZrianinaMariia** [comment]
:cold_sweat: something doesn't look right. Take a look at pagination.

File: src/views/RepoPage/RepoTable.vue (hunk 1 above)
Comment (Ada-siqiliu) [reply src/views/RepoPage/RepoTable.vue]: Hi,I have modified all problems,please review agian. Thx

**Comment by Ada-siqiliu** [comment]
> 😰 something doesn't look right. Take a look at pagination.

Hi,I have modified all problems,please review agian. Thx

**Comment by lee-chase** [comment]
:thumbsup: looks good, you are one step closer to earning the Carbon Developer Essentials Vue badge

**Pull request closed without merging**